sequential/maps/README.md

```diff
@@ -46,3 +46,46 @@ to_map([]).
 %% #{}
 ```
 [solution](src/solution/list_to_map.erl).
+
+### Records to Maps
+
+Create a record named `person` that has the attributes `name` and `age`. Then write a function `map:records_to_maps(Records)` that converts a list of records (in this case people) into a list of maps with the attributes. For this you should use `lists:map`.
+
+Example:
+```erlang
+map:records_to_maps([]).
+%% []
+
+map:records_to_maps([#person{name="Pepe", age=28}, #person{name="Luis", age=77}]).
+%% [#{age => 28,name => "Pepe"},#{age => 77,name => "Luis"}]
+```
+
+[solution](src/solution/records_to_maps.erl).
+
+### Maps to Records
+
+Create a record named `person` that has the attributes `name` and `age`. Then write a function `map:maps_to_records(Records)` that converts a list of maps into a list of records (in this case people). For this you should only use recursion.
+
+Example:
+```erlang
+map:maps_to_records([]).
+%% []
+
+map:maps_to_records([#{age => 28,name => "Pepe"},#{age => 77,name => "Luis"}]).
+%% [#{age => 77,name => "Luis"}, #{age => 28,name => "Pepe"}]
+```
+
+[solution](src/solution/maps_to_records.erl).
+
+### Proplist to Map
+
+Write a recursive function `proplist_to_map/1` that takes a proplist (a list of tuples) and builds a map from it. Use the first component of each tuple as the key and the second component as the value.
```

Comment:
One question. Not sure if you came up with this problem or someone else did, but you do know there are Erlang functions to do this kind of work, right? (e.g. `maps:from_list`, `maps:to_list`)

If the idea is to implement them "by hand" without relying in the standrad library, I would at least add a disclaimer in the readme stating that in a real world scenario you would use the standard erlang functions instead of reimplementing them.

Reply:
Yeah the idea was to implement  "by hand". I'll add the disclaimer.